crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -504,6 +562,7 @@ type Query {
 	transactionBlock(digest: String!): TransactionBlock
 	checkpointConnection(first: Int, after: String, last: Int, before: String): CheckpointConnection
 	transactionBlockConnection(first: Int, after: String, last: Int, before: String, filter: TransactionBlockFilter): TransactionBlockConnection
+	eventConnection(first: Int, after: String, last: Int, before: String, filter: EventFilter!): EventConnection
```

Comment:
Is the event filter mandatory, it seems to be optional.

Reply:
since im using the indexer impl. yes